source/style-guide/markup/directives/metadata.txt

```diff
@@ -7,18 +7,23 @@ Page Metadata Directives
 meta
 ----
 
-Use ``.. meta::`` to add |html| meta tags to a page. Adding meta tags 
-aids in searches both internally and through the Internet. You can add 
-both keywords and descriptions. To learn more, see 
+Use ``.. meta::`` to add |html| meta tags to a page. Adding meta tags
+aids in searches both internally and through the Internet. You can add
+both keywords and descriptions. To learn more, see
 :ref:`seo-guidelines`.
 
+.. important::
+
+   MongoDB Technical Writers should use the meta keyword standards defined in
+   the "Taxonomy Tagging Instructions" internal wiki page.
```

Comment:
Can we link?

Reply:
I'm not sure if we can link to the internal wiki since the meta site is available to the public. I might be mistaken though.

Reply:
@corryroot I avoided linking because the wiki is only accessible by MongoDB employees, and the style guide is public. Let me know if you think it should be linked or if you have any other solutions in mind.